src/components/icons/svgs/CircleIcon.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+import withIconSvg from '../helpers/withIconSvg';
+
+const Icon: React.FC = (props) => (
+	<svg
+		height="8"
+		viewBox="0 0 8 8"
+		width="8"
+		{...props}
+	>
+		<circle cx="4" cy="4" r="4"/>
+	</svg>
+);
+
+export default withIconSvg(
+	Icon,
+	true,
+	{
+		tags: [
+			'round',
```

Comment:
Thoughts on adding `circle` to the list?

Reply:
Yeah, something I didn't document was that search checks the filename, import name, and these tags. But I like the idea of tags including everything, so this was a miss on my part. Thanks for catching it! I'll make the change.

Reply:
Aha! That makes sense. In that case, I'm okay either way here!

Reply:
You know, I really like the idea of doing it for two reasons:

1. more explicit
2. if the icon is renamed at any point, for any reason, this keyword is still captured

Thanks for the feedback!